server/src/test/java/io/crate/protocols/ssl/SslContextProviderTest.java

```diff
@@ -100,6 +99,12 @@ public void testClassLoadingWithValidConfiguration() {
         assertThat(sslContext.cipherSuites(), not(empty()));
     }
 
+    @Test
+    public void test_no_truststore_gets_defaults_certs() {
+        TrustManager[] trustManagers = SslContextProvider.createTrustManagers(null);
+        assertThat(SslContextProvider.getDefaultCertificates(trustManagers).length, greaterThan(0));
```

Comment:
I'm not sure it's always guaranteed (if cacerts and all default files get removed?) but probably not likely